region/lowernorfair/east/Wasteland.json

```diff
@@ -1512,6 +1516,7 @@
         {"heatFrames": 240}
       ],
       "clearsObstacles": ["B"],
+      "flashSuitChecked": true,
```

Comment:
This strat can break the block with blue without losing flash suit, and its not really much harder.